src/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import { Provider } from 'react-redux';
+import store from './redux/store';
+import App from './App';
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+  <React.StrictMode>
+    <Provider store={store}>
+      <App />
```

Comment:
Hello @cynthiainga Thanks for the feedback. What do you think is behind this, please? I personally think for you to be able to access and run this locally, you'd need to create an API key for yourself. I can include this instruction in the README file so that it is clear. If there's an alternative approach, I'd be happy to learn that. Kindly let me know.